src/byteweight/options.ml

```diff
@@ -0,0 +1,35 @@
+open Core_kernel.Std
```

Comment:
nope, either move byteweight to the symbols, or do not use symbols. But never copy-paste code.